_docs/tasks/security/authn-policy.md

```diff
@@ -88,6 +86,31 @@ NAME          AGE
 enable-mtls   1m
 ```
 
+Add this destination rule to config client side to use mTLS:
+
+```bash
+cat <<EOF | istioctl create -f -
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "enable-mtls"
+  namespace: "foo"
+spec:
+  host: *.foo.svc.local.cluster
+  trafficPolicy:
+    tls:
+      mode: MUTUAL
+      clientCertificate: /etc/certs/cert-chain.pem
+      privateKey: /etc/certs/key.pem
+      caCertificates: /etc/certs/root-cert.pem
+EOF
+```
+
+> Note:
+* This rule is based on the assumption that there is no other destination rule in the system. If it's not the case, you need to modify traffic policy in existing rule accordingly.
+* `*.foo.svc.local.cluster` matches all services in namespace `foo`.
+* You can also use the mode `ISTIO_MUTUAL` and let Istio to assign the path for key and certificates automatically.
```

Comment:
You might want to make this more specific. If people are doing mTLS on a specific port, then you can customize the destination rule for that port alone. I think that field is called portTrafficPolicy in the destination rule

Reply:
This DR is to match with the above authn policy, that is not port specific. I have another port-specfic authn policy example below, with portLevelSettings DR